HelpMyStreetFE/HelpMyStreetFE/Services/Users/IAuthService.cs

```diff
@@ -11,7 +11,7 @@ public interface IAuthService
         Task LoginWithTokenAsync(string token, HttpContext httpContext);
         Task LoginWithUserId(int userId, HttpContext httpContext, CancellationToken cancellationToken);
         Task<string> VerifyIdTokenAsync(string token);
-        Task<User> GetCurrentUser(HttpContext httpContext, CancellationToken cancellationToken);
+        Task<User> GetCurrentUser(CancellationToken cancellationToken);
```

Comment:
Would be nice to make the same change for the other functions here taking an HttpContext...